capa/rules/__init__.py

```diff
@@ -194,6 +196,59 @@ def __repr__(self):
         return str(self)
 
 
+# COM data source https://github.com/stevemk14ebr/COM-Code-Helper/tree/master
+VALID_COM_TYPES = {
+    "class": {"db_path": "assets/classes.json.gz", "prefix": "CLSID_"},
+    "interface": {"db_path": "assets/interfaces.json.gz", "prefix": "IID_"},
+}
+
+
+def translate_com_feature(com_name: str, com_type: str) -> ceng.Or:
```

Comment:
```suggestion
def translate_com_feature(com_name: str, com_type: Literal["class", "interface"]) -> ceng.Or:
```

alternatively introduce an enum for this.

Reply:
not blocking, just a suggestion.

Reply:
I have defined an enum class for `com_type`, but I think it looks better by passing it as just a string.
What are your thoughts @williballenthin!!

Reply:
using an enum is more consistent with other parts of capa, though i agree that just a string looks better on its own. lets use the enum for consistency here.